cmd/ocm/tunnel/cmd.go

```diff
@@ -42,6 +46,9 @@ var Cmd = &cobra.Command{
 }
 
 func init() {
+	flags := Cmd.Flags()
+
+	flags.BoolVar(&args.short, "short",false, "to shorten the 'sshuttle' path")
```

Comment:
```suggestion
	flags.BoolVar(&args.short, "short", false, "to shorten the 'sshuttle' path")
```
to continue the coding-in-browser streak :wink:  I think this will fix the lint error.